contactql/parser.go

```diff
@@ -204,7 +212,7 @@ func (q *ContactQuery) String() string {
 }
 
 // ParseQuery parses a ContactQL query from the given input
-func ParseQuery(text string, redaction envs.RedactionPolicy) (*ContactQuery, error) {
+func ParseQuery(text string, redaction envs.RedactionPolicy, fieldResolver func(string) assets.Field) (*ContactQuery, error) {
```

Comment:
I'm fine with this but can we declare a type for this instead of having it inline? IE:
`type FieldResolverFunc func(string) assets.Field`

Reply:
updated